packages/graphql/src/graphql/directives/fulltext.ts

```diff
@@ -43,6 +45,9 @@ export const fulltextDirective = new GraphQLDirective({
                             fields: {
                                 type: new GraphQLNonNull(new GraphQLList(GraphQLString)),
                             },
+                            queryName: {
+                                type: GraphQLString,
+                            }
```

Comment:
Hmmmmm, just had a thought of could we add `indexName` now, make it take precedence over `name` and mark `name` as deprecated. Means the only thing to do for 4.0 will be remove it, and some will already be migrated!

```suggestion
                            queryName: {
                                type: GraphQLString,
                            }
                            indexName: {
                                type: GraphQLString,
                            }

```